release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -25,7 +25,7 @@ You must use {op-system} machines for the control plane, and you can use either
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
 //TODO: Add this for 4.14
-Starting with {product-title} 4.12, an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Starting with {product-title} 4.14, the Extended Update Support (EUS) phase for even numbered releases has been extended from 24 months to 36 months. For odd numbered releases, the EUS phase remains at 18 months. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```

Comment:
I think it needs to be clearer that the additional year is an optional additional offering. It is not part of the base lifecycle and is in fact an additional EUS term. See the lifecycle page for relevant phrasing.

Reply:
Thanks, updated.